extensions/amp-a4a/0.1/test/test-a4a-integration.js

```diff
@@ -215,6 +215,31 @@ describe('integration test: a4a', () => {
         });
       });
 
+  it('should collapse slot when creative response has code 204', () => {
+    headers = {};
+    headers[SIGNATURE_HEADER] = validCSSAmp.signature;
+    mockResponse = {
+      arrayBuffer: () => utf8Encode(validCSSAmp.reserialized),
+      bodyUsed: false,
+      headers: new FetchResponseHeaders({
+        getResponseHeader(name) {
+          return headers[name];
+        },
+      }),
+      status: 204,
+    };
+    xhrMock.withArgs(TEST_URL, {
+      mode: 'cors',
+      method: 'GET',
+      credentials: 'include',
+    }).onFirstCall().returns(Promise.resolve(mockResponse));
+    const forceCollapseStub =
+        sandbox.stub(MockA4AImpl.prototype, 'forceCollapse');
+    return fixture.addElement(a4aElement).then(unusedElement => {
+      expect(forceCollapseStub).to.be.calledOnce;
+    });
+  });
```

Comment:
Are there already cases for `!fetchResponse` and `!fetchResponse.arrayBuffer`?  If not, it's worth adding them here.  (And if there are, being sure that they invoke the no-content logic.)